components/brave_shields/browser/ad_block_service.cc

```diff
@@ -154,6 +156,21 @@ base::Optional<base::Value> AdBlockService::HiddenClassIdSelectors(
     hide_selectors = std::move(regional_selectors);
   }
 
+#if !defined(OS_ANDROID) && !defined(CHROME_OS)
```

Comment:
we don't need OS guards around this, `OS_ANDROID` should just default to enabled

Reply:
I will get rid of that part at all, just didn't want to make it in that PR to save time as need to correct native part after modifications